MoltenVK/MoltenVK/API/mvk_datatypes.h

```diff
@@ -378,10 +378,13 @@ MTLMultisampleStencilResolveFilter mvkMTLMultisampleStencilResolveFilterFromVkRe
 #endif
 
 /** Returns the Metal MTLViewport corresponding to the specified Vulkan VkViewport. */
-MTLViewport mvkMTLViewportFromVkViewport(VkViewport vkViewport);
+MTLViewport mvkMTLViewportFromVkViewport(const VkViewport vkViewport);
```

Comment:
Why the `const` on a struct passed by value?

Reply:
Good point. Fixed.